datasets/overlastgebieden/overlastgebieden.json

```diff
@@ -55,7 +55,15 @@
           "oov_naam": {
             "type": "string",
             "description": "Naam overlastgebied zoals gedefinieerd door Openbare Orde en Veiligheid (OOV)."
-          }
+          },
+          "geldigheid": {
```

Comment:
Zijn de veldnamen wellicht nog wat specifieker te maken? Bijv. "dagenGeldig"? En "tijd" is dat uren/maanden/dagen?

Reply:
Ja goed punt, ik heb het explicieter gemaakt en de beschrijving aangevuld. Is zo genoemd om in overeenstemming te zijn met een gerelateerde dataset t.a.v. covid19.